### ISSUE TITLE
Several binary operators of VB is not represented in IOperation

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn#9174]
`^` (power), `Like` and `&` (concat) are not a part of `IOperation`.

Here's how to repro this bug:
- In the case of user defined operator, it will crash in `BoundUserDefinedBinaryOperator.get_IBinaryOperatorExpression_BinaryOperationKind()` with message `Unexpected value 'Power' of type 'Microsoft.CodeAnalysis.VisualBasic.BinaryOperatorKind'
  `
- In the case of none user-defined operator, its `BinaryOperationKind` property will have value `Invalid`
  (returned by `Expression.DeriveBinaryOperationKind` method`

``` vb
Public Class B2
    Public Shared Operator ^(x As B2, y As B2) As B2 
        System.Console.WriteLine("^")
        Return x
    End Operator

    Public Shared Operator Like(x As B2, y As B2) As B2
        System.Console.WriteLine("Like")
        Return x
    End Operator

    Public Shared Operator &(x As B2, y As B2) As B2 
        System.Console.WriteLine("&")
        Return x
    End Operator
End Class

Module Module1
    Sub Main() 
        Dim x, y As New B2()
        Dim r As B2

        r = x ^ y
        r = x Like y 
        r = x & y    

        dim i = 2 ^ 3
    End Sub
End Module
```

**Comment by srivatsn** [comment]
Need to keep in 1.2 to fix the crash.

**Comment by genlu** [comment]
The crash is fixed by #9310. Moving this to 1.3 to keep track implemention of missing operators in VB.

**Comment by jinujoseph** [comment]
this is done
